index.ts

```diff
@@ -1 +1 @@
-export { YourWebComponent } from './src/your-webcomponent';
+export { ReviewForm } from './src/review-form';
```

Comment:
Do we have a use-case for exporting the `Review` model also? ie a service that fetches reviews. It may be premature if not (and can be exported later), it just popped into my head.